Mexicali-Mexico.md

```diff
@@ -0,0 +1,5 @@
+# Mexicali, Mexico
+
+### WHere to eat
+
```

Comment:
Hi @licjapodaca,

Just want to know where's good place to go for some eats! :smile: